config/initializers/voxpupuli.rb

```diff
@@ -11,3 +11,7 @@
 CENTOS_SUPPORT_RANGE  = VOXPUPULI_CONFIG.dig('support_ranges', 'centos')
 FREEBSD_SUPPORT_RANGE = VOXPUPULI_CONFIG.dig('support_ranges', 'freebsd')
 FEDORA_SUPPORT_RANGE  = VOXPUPULI_CONFIG.dig('support_ranges', 'fedora')
+
+REPOSITORY_STATUS_CHECKS = []
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Freeze mutable objects assigned to constants.](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5444834)